components/k8s-model-server/http-proxy/server.py

```diff
@@ -19,10 +19,12 @@
 from itertools import repeat
 import base64
 import logging
-import grpc
+import random
```

Comment:
If we do it in the http server proxy we won't get this for gRPC requests; right?